src/layman/layer/filesystem/gdal.py

```diff
@@ -238,7 +238,10 @@ def is_nodata_out_of_min_max(filepath, *, nodata_values):
 
 def is_normalized_alpha_needed(filepath, *, color_interp, nodata_values):
```

Comment:
Probably keep previous version and remove the alpha band in compress_and_mask_raster_file_async using `-b 1` only if it is Float data type